### DISCUSSION TITLE
More recent flatpak release

### DISCUSSION THREAD

**Discussion by modcolocko** [andyholmes/valent#711] [Distribution Packaging]
From what I can tell, the last Flatpak release on the Nightly is from March 31. Is there any chance we could get a release that includes all the updates since then, instead of having to build ourselves (especially considering there aren't build instructions)

Thanks, the project is greatly appreciated.

**Comment by andyholmes** [comment]
Hi, the Flatpak repo on the front-page and README (https://valent.andyholmes.ca/repo) is updated on every commit to the `main` branch. So if you're using that, then you have the latest.

The existing tag (`1.0.0-alpha.45` IIRC) was more of a formality to get some CI checks running and so forth. I still consider the project effectively `0.1.0`, but the needle will probably go directly from that to `1.0.0-beta` without much warning.

I plan on tagging another alpha to mark the GNOME 47 platform release, but that again will be nominal. The latest can just be grabbed from the nightly Flatpak repo.

**Comment by modcolocko** [comment]
Hmm, removing and reinstalling valent seems to get a more recent release, however now the option for messaging is not present.

**Reply by andyholmes** [reply]
The SMS plugin was rewritten and can be found in the primary window menu for now.
